src/main/java/me/magnet/consultant/ServiceIdentifier.java

```diff
@@ -123,6 +138,7 @@ public String toString() {
 		getDatacenter().ifPresent(dc -> descriptors.add("dc=" + dc));
 		getHostName().ifPresent(host -> descriptors.add("host=" + host));
 		getInstance().ifPresent(instance -> descriptors.add("instance=" + instance));
+		descriptors.add("tags=[" + getTags().stream().collect(Collectors.joining(",")) + "]");
```

Comment:
Although without brackets it might be a bit hard to parse. Has this broken any tests related to reading config?